### PR TITLE
[wendelinwemhoener] iP

### PR DESCRIPTION
**Pull request by wendelinwemhoener**

### CODE REVIEW

File: src/main/java/careo/Duke.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        printHorizontalLine();
+        for (Task task : tasks) {
+            System.out.println("    " + counter + "." + task);
+            counter++;
+        }
+        printHorizontalLine();
+    }
+
+
+    /**
```
Comment (charkty) [inline src/main/java/careo/Duke.java]: Very clear and concise comment that follows the correct form

File: src/main/java/careo/Event.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package careo;
+
+public class Event extends Task {
+    /** When the event starts */
```
Comment (charkty) [inline src/main/java/careo/Event.java:7]: very short and clear naming of variables, all in the right format too

File: src/main/java/careo/Duke.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+     * Prints a line of dashes indented by four spaces.
+     */
+    private static void printHorizontalLine() {
+        System.out.println("    " + "-".repeat(76));
+    }
```
Comment (MrOPPA1) [inline src/main/java/careo/Duke.java]: It's really cool to use a funciton to print a horizontal line and i love the design of using the function of "repeat".

File: src/main/java/careo/Event.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + from + " to: " + to + ")";
+    }
```
Comment (MrOPPA1) [inline src/main/java/careo/Event.java:41]: maybe u can define a variable(protected String for example) in the base class Task(suppose to call it "type"),and in the toString function of Task just use "["+type+"]",and change the type in the constructer of each class~

**Review by charkty — COMMENTED** [review]
Generally, your code is very neat and readable. A slight improvement can be made on reducing some chunks of code by perhaps creating more functions

**Review by madhanse — COMMENTED** [review]
Good Job.

**Pull request closed without merging**